jenkins-builds/common-SLES12-2016.10

```diff
@@ -14,4 +14,13 @@ likwid-4.1.2.eb
 HDFView-2.13.0.eb
 ctags-5.8.eb
 lynx-2.8.8rel.2.eb
-tmux-2.3.eb
\ No newline at end of file
+tmux-2.3.eb
+# --- scorep
+zlib-1.2.8-scorep.eb
```

Comment:
> we need to change all jenkins projects.

why ? just need modulepath to point to /apps/common/UES/SLES12/easybuild/modules/all